src/NUnitFramework/framework/Internal/Commands/OneTimeTearDownCommand.cs

```diff
@@ -71,6 +71,9 @@ public override TestResult Execute(TestExecutionContext context)
                 IDisposable disposable = context.TestObject as IDisposable;
                 if (disposable != null && Test is IDisposableFixture)
                     disposable.Dispose();
+
+                //Clear references to test objects to reduce memory usage
+                context.TestObject = null;
```

Comment:
I'm suspicious. If this is necessary, then that must mean that the context isn't being properly disposed.